test/cases/attrs-spread.pug

```diff
@@ -0,0 +1,5 @@
+div(...attrs)
+div(...attrs=foo)
```

Comment:
My thinking was that it could mean you can namespace things:  e.g.

```
div(...data=myDataAttributes)
```

compiled with:

```js
{myDataAttributes: {foo: 'foo-val', bar: 'bar-val'}}
```

results in:

```html
<div data-foo="foo-val" data-bar="bar-val"></div>
```

Probably this is the wrong syntax for that though.  I think I would prefer to support something like:

```
div(data=...myDataAttributes)
```
or
```
div(data...=myDataAttributes)
```

as I think that reads better/more clearly.  Probably best to remove support for having a "value" for spread attributes.  If we make sure it's a syntax-error for now, we can always add it in later.